frontend/src/routes/auth/+page.svelte

```diff
@@ -42,19 +47,11 @@
       </h1>
     </div>
 
-    {#if $user === null}
-      <div class="pt-4">
-        <Button
-          class="flex min-w-[140px] space-x-2"
-          variant="secondary"
-          type="button"
-          on:click={login}
-        >
-          <Google class="w-4" />
-          <span>Login to Decrypt</span>
-        </Button>
-      </div>
-    {:else}
+    <div class={$user === null ? "" : "hidden"}>
+      <div id="login-button"><Skeleton class="h-10 w-full" /></div>
```

Comment:
If you add these styles, it will make it less awkward.

```suggestion
      <div id="login-button" style="color-scheme:light">
            <Skeleton class="h-10 w-[200px]" />
      </div>
```


<img src="https://github.com/shadanan/cipherly/assets/6494584/088b42a8-fea6-4747-bf21-703c8af0550a" width="400"/>
<img src="https://github.com/shadanan/cipherly/assets/6494584/d335af2e-4a06-412c-b546-b5087a072a8f" width="400"/>

Reply:
Brilliant!